### PR TITLE
Update tbench references to yab, add content to README

### PR DESCRIPTION
**Pull request by prashantv**
@abhinav @akshayjshah

### CODE REVIEW

**Comment by prashantv** [comment]
cc @breerly

File: README.md [hunk 1]
```diff
@@ -1,5 +1,99 @@
-tbench
+yab
 ======
 
-tbench is a tool to benchmark Thrift services. It currently supports
-TChannel as a transport, but will also support HTTP.
+`yab` (Yet Another Benchmarker) is tool to benchmark Thrift services. It has
+support for both HTTP and TChannel services.
```
Comment (abhinav) [inline README.md]: Doesn't *quite* support HTTP Thrift services.

Comment (prashantv) [inline README.md]: True, added a note

File: README.md [hunk 2]
```diff
@@ -1,5 +1,102 @@
-tbench
+yab
 ======
 
-tbench is a tool to benchmark Thrift services. It currently supports
-TChannel as a transport, but will also support HTTP.
+`yab` (Yet Another Benchmarker) is tool to benchmark Thrift services. It has
```
Comment (breerly) [inline README.md]: Perhaps: "to benchmark YARPC services".

Then you can just add `raw` and `json` to the TODO :)

**Comment by breerly** [comment]
LGTM :shipit:

**Comment by abhinav** [comment]
:+1:

**Pull request merged**